src/Network/DogStatsd.hs

```diff
@@ -0,0 +1,53 @@
+module Network.DogStatsd (
+ dogStatsdClient
+, increment
+, decrement
+, count
+, gauge
+, timing
+, histogram
+) where
+
+import Network.Statsd (Stat, Type(..), fmtDatagram)
+import Network.Statsd.UdpClient(UdpClient(..), fromURI, send)
+
+import Control.Monad
+import Data.Maybe
+import Data.List
+import Data.Time.Units
+import Text.Printf
+import Network.URI
+
+type Tags = [(String, String)]
+
+dogStatsdClient :: String -> IO UdpClient
+dogStatsdClient url = (fromURI . fromJust . parseURI) url
+
+increment :: UdpClient -> Stat -> Tags -> IO ()
+increment client stat = count client stat 1
+
+decrement :: UdpClient -> Stat -> Tags -> IO ()
+decrement client stat = count client stat (-1)
+
+count :: UdpClient -> Stat -> Int -> Tags -> IO ()
+count client stat value tags = void . send client $ fmtDogStatsdDatagram stat value Count tags
```

Comment:
So I don't think point free notation works here because the function we are calling is really `void . send` which only takes a single (String) argument. The tags argument needs to be applied to the `fmtDogStatsdDatagram`.

There might be a way to do this that I don't know about, though! I'd love to learn.